src/registry_schemas/example_data/schema_data.py

```diff
@@ -762,7 +762,7 @@
                 'lastName': 'Party',
                 'middleName': 'P',
                 'email': 'cp@example.com',
-                'orgName': '',
+                'organizationName': '',
```

Comment:
I changed `orgName` to `organizationName` because in schema.org it usually doesn't use truncate names and our DB column is `organization_name`.

Reply:
Sounds good.